priv/repo/migrations/20160520141230_add_token_to_member.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Community.Repo.Migrations.AddTokenToMember do
+  use Ecto.Migration
+
+  def change do
+    execute "TRUNCATE TABLE members;"
```

Comment:
Oh yeah great point. Since you’ve done it elsewhere that’s good to keep it consistent.


One thing to keep in mind for the future is that the (main?) reason to change the secret key for encrypting the member id is if the key is compromised. The same could be said for tokens in the db. If they’re compromised you’d have to reset those and the links would no longer work.

> On May 20, 2016, at 2:16 PM, Jason Draper <notifications@github.com> wrote:
> 
> In priv/repo/migrations/20160520141230_add_token_to_member.exs <https://github.com/drapergeek/community/pull/51#discussion_r64084665>:
> 
> > @@ -0,0 +1,11 @@
> > +defmodule Community.Repo.Migrations.AddTokenToMember do
> > +  use Ecto.Migration
> > +
> > +  def change do
> > +    execute "TRUNCATE TABLE members;"
> That is really cool!
> 
> My only issue is if you decide to cycle the secret key, all the users now have no way to edit their things. I'm not sure that's ideal for long term token type things.
> 
> I may consider that in a refactoring but this i the pattern I used for jobs so I think for now I'll stick with consistency.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/51/files/e54cd1ea5f5012fd917d7038dfe85fe09d120eac#r64084665>